docs/google.rst

```diff
@@ -130,6 +130,7 @@ google.lifeSciences.debug                      When ``true`` copies the `/google
 google.lifeSciences.preemptible                When ``true`` enables the usage of *preemptible* virtual machines or ``false`` otherwise (default: ``true``)
 google.lifeSciences.usePrivateAddress          When ``true`` the VM will NOT be provided with a public IP address, and only contain an internal IP. If this option is enabled, the associated job can only load docker images from Google Container Registry, and the job executable cannot use external services other than Google APIs (default: ``false``). Requires version ``20.03.0-edge`` or later.
 google.lifeSciences.network                    Set network name to attach the VM's network interface to. The value will be prefixed with global/networks/ unless it contains a /, in which case it is assumed to be a fully specified network resource URL. If unspecified, the global default network is used. Requires version ``21.03.0-edge`` or later.
+google.lifeSciences.serviceAccountEmail        Define the email of the service account to install on the VM. If not specified, the default Compute Engine service account for the project will be used.
```

Comment:
I saw few other fields have `Requires version ``21.03.0-edge`` or later.`
Not sure if I need similar things, and if yes, what would be the value?

Reply:
Ok, we'll merge this into `21.05.0-edge`. You may want to add it. Also, maybe the following reads better 

```
Define the Google service account email to use for the pipeline execution. If not specified, the default Compute Engine service account for the project will be used.
```

Reply:
Done. Thanks @pditommaso for reviewing again.  I think this PR is ready to go?